test/commands/scanner/run.test.ts

```diff
@@ -715,7 +716,14 @@ describe('scanner:run', function () {
 				expect(ctx.stdout).to.contain(runMessages.getMessage('output.filtersIgnoredCustom'));
 			});
 
-
+		setupCommandTest
```

Comment:
Aside from the variable renaming, I had to add a bit more test coverage since we dropped below the acceptable minimum.